### ISSUE TITLE
Create manual lock and remove specific lock

### ISSUE THREAD

**Issue by Wouter0100** [restic/restic#3054]
Output of `restic version`
--------------------------
```
restic 0.10.0 compiled with go1.15.2 on linux/amd64
```

What should restic do differently? Which functionality do you think we should add?
----------------------------------------------------------------------------------
restic in itself shouldn't do anything differently, but it would be awesome to allow us to manually create locks using a command and remove specific locks with `unlock`.

What are you trying to do? What problem would this solve?
---------------------------------------------------------
This feature would allow us to start a backup process (which could make multiple snapshots) on multiple servers at, for example, 1AM. Each server creating its own lock, and removing the lock after all snapshots are made. 

Later in the night, at like 2AM, we can start a cronjob to watch the locks. If the repository is fully unlocked, we can safely start a forget with prune. 

This would be possible with the current way build-in locks work, but not without a workaround: currently locks work only when a snapshot is being made. But the time my script is doing different things (for example, preparing the backup), no lock exists in restic. A solution for this would be to wait 5 minutes (or a certain amount to be sure no new snapshots come in) on the backup server before starting to forget and prune, but I see this as a workaround to a feature that would be awesome to have in restic. 

Did restic help you today? Did it make you happy in any way?
------------------------------------------------------------
I've been using restic for [over a year now](https://forum.restic.net/u/Wouter0100/activity/topics), and I couldn't be happier with it. I made many integrations with it, and our whole backup infrastructure runs purely on restic. From VM's, webhosting, and so on. Seeing #2718 getting ready for merge, would (nearly?) makes me kneel for this awesome community :-). #hyped

**Comment by fd0** [comment]
Thanks for the idea, I'm closing it as a duplicate of #2166 :)
